.github/workflows/qc.yml

```diff
@@ -26,12 +26,11 @@ jobs:
       - name: Install dependencies
         run: poetry install --no-interaction
 
+      # - name: Check common spelling errors
+      #   run: poetry run tox -e codespell
+
       - name: Check code quality with flake8
-        run: poetry run tox -e flake8
-#       - name: Check package metadata with Pyroma
-#         run: poetry run tox -e pyroma
-      - name: Check static typing with MyPy
-        run: poetry run tox -e mypy
+        run: poetry run tox -e lint
```

Comment:
Why is the goal called "Check code quality with flake8" and runing lint (ie. black) instead of flake8? Why is mypy removed?

Reply:
typo in cookiecutter... will correct it